src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.MoveFile.cs

```diff
@@ -0,0 +1,238 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Threading;
+using Xunit;
+
+public partial class FileSystemWatcher_4000_Tests
+{
+    #region WindowsTests
+
+    [Theory]
+    [InlineData(WatcherChangeTypes.Changed, "changed", false)]
+    [InlineData(WatcherChangeTypes.Created, "created", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "deleted", false)]
+    [InlineData(WatcherChangeTypes.Deleted, "changed", false)]
+    [InlineData(WatcherChangeTypes.Renamed, "renamed", true)]
+    [InlineData(WatcherChangeTypes.Renamed, "changed", true)]
```

Comment:
There's an entry that's Deleted - "changed" and one that's Renamed - "changed", whereas all the others match up.  Is that on purpose?  Also, why not just use type.ToString() as the typeName?